packages/@react-types/list/src/index.d.ts

```diff
@@ -21,7 +21,6 @@ import {
   SpectrumSelectionProps,
```

Comment:
I would love to delete the `@react-types/list` package now that aria/spectrum define and export the types but a bit worried about if people are using it since it have been released.

Reply:
would it be weird to change this package to re-export the types from aria/spectrum instead? only for backward compatibility I guess. not sure how important tho

Reply:
feels a bit weird to have the dependencies go the other way, but at least it wouldn't be circular since nothing else should depend on this types package. I'll go ahead and update